doc/api/esm.md

```diff
@@ -309,6 +309,36 @@ modules it can be used to load ES modules.
 The `import.meta` meta property is an `Object` that contains the following
 properties.
 
+### `import.meta.dirname`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+> Stability: 1.2 - Release candidate
+
+* {string} The directory name of the current module. This is the same as the
+  [`path.dirname()`][] of the [`import.meta.filename`][].
+
+> **Caveat** only local modules support this property. Modules not using the
+> `file:` protocol will not provide it.
+
+### `import.meta.filename`
```

Comment:
I still agree with others that `filename` is misleading, especially considering that other runtimes implement `import.meta.file` as the actual file name and `import.meta.path` as the full path. (Then again, this entire API is a questionable fit for ESM I suppose.)

I assume this has been discussed and dismissed elsewhere, I am only mentioning it in case it hasn't.

Reply:
It was discussed in the spec proposal. I summarized the conclusions here: https://github.com/wintercg/import-meta-registry/blob/main/import-meta-path-helpers.md#alternatives-considered